SelfTableViewManager/Classes/SelfTableViewManager.swift

```diff
@@ -251,11 +251,8 @@ extension SelfTableViewManager {
         
         allRows.removeObjects(in: discardRows)
         
-        if #available(iOS 11.0, *) {
-            performBatchUpdatesRemoveRows(paths: paths, allRows: allRows, animation: animation)
-        } else {
-            performBeginUpdatesRemoveRows(paths: paths, allRows: allRows, animation: animation)
-        }
+        rows = allRows as [AnyObject]
```

Comment:
I could not find a way to test it unitary, so if anyone has any idea, feel free to suggest 😁